tests/test_analysis_nanopore.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
```

Comment:
OMG YOU HAVE TESTS!

Reply:
Haha, yes 😄 I worked all spring to bring the test coverage from ~41% to a whopping 86% 😉